include/mixer.h

```diff
@@ -175,9 +190,9 @@ class MixerChannel {
 	void Enable(bool should_enable);
 	void FlushSamples();
 
-	float volmain[2] = {1.0f, 1.0f};
-	std::atomic<int> done = 0; // Timing on how many samples have been done
-	                           // by the mixer
+	AudioFrame volmain = {1.0f, 1.0f};
+	std::atomic<int> frames_done = 0; // Timing on how many sample frames
+	                                  // have been done by the mixer
```

Comment:
Huge readability improvement; `frames_done` actually has a unit now that implies a quantity, where as "done" is often understood to be a binary state: "Are you done or not?".